packages/slate-hyperscript/src/creators.ts

```diff
@@ -36,6 +36,7 @@ const resolveDescendants = (children: any[]): Descendant[] => {
       const c = child // HACK: fix typescript complaining
 
       if (
+        prev &&
```

Comment:
Not strictly necessary as we don't use this module, just keeping the test passing